src/modules/emote_menu/components/Emote.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from '../styles/emote.module.css';
+
+function Emote({emote, onClick, onMouseOver}) {
+  return (
+    <button
+      key={emote.code}
+      onClick={() => onClick(emote)}
+      onMouseOver={() => onMouseOver(emote)}
+      onFocus={() => onMouseOver(emote)}
+      type="button"
+      className={styles.emote}>
+      <img
+        className={styles.emoteImage}
+        srcSet={`${emote.images['1x']} 1x, ${emote.images['2x']} 2x, ${emote.images['3x']} 4x`}
```

Comment:
![image](https://user-images.githubusercontent.com/2942184/131236835-d108e7d1-5f73-4a1d-9651-46f6eab76982.png)
![image](https://user-images.githubusercontent.com/2942184/131236838-ff6c5952-e298-4b13-b9c2-8f76f36f87d6.png)

Reply:
can we also show a loading state of some kind?

Reply:
search is also for some reason unpredictable with rendering urls 
![image](https://user-images.githubusercontent.com/2942184/131237080-cdcfbb31-795c-4036-a31f-7b76e9c26a2d.png)